### DISCUSSION TITLE
On the waitlist but my email is Undeliverable.

### DISCUSSION THREAD

**Discussion by DinhHuynhTienPhu** [community/community#8289] [Copilot Conversations]
About serveral months ago i joined the waitlist. My account setting is like this: 
![ảnh](https://user-images.githubusercontent.com/40557558/144194190-5b50168e-43f7-4c9c-b80b-952407c817c3.png)
Unknowing the email setting was "Undeliverable", i just waited. But my friend recently told me that he just wait for about a week in the waitlist and then received mail from copilot team. So i go back and just change my email to
![Capture](https://user-images.githubusercontent.com/40557558/144195042-e806cfd0-fd70-48da-8042-2313e0770974.PNG)
Now i think that maybe the copilot team has sent me the mail already? Will they send the mail again or i wont ever receive mail again and wait in the waitlist forever? :(((

**Comment by davecheney** [comment]
The short answer is install the copilot extension. If it works, you got an invite but it was lost in the mail.
